### ISSUE TITLE
Error with ShellToSolidShellProcess and pybind11 - Windows

### ISSUE THREAD

**Issue by PhilippeBussetta** [KratosMultiphysics/Kratos#1868]
I cannot compile the master branch under windows because of link error

```
1>add_custom_utilities_to_python.cpp
1>Info: Boost.Config is older than your compiler version - probably nothing bad will happen - but you may wish to look for an update Boost version.  Define BOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE to suppress this message.
2>   Creating library D:/Kratos-FeatureElement/cmake_build/applications/StructuralMechanicsApplication/Release/KratosStructuralMechanicsApplication.lib and object D:/Kratos-FeatureElement/cmake_build/applications/StructuralMechanicsApplication/Release/KratosStructuralMechanicsApplication.exp
2>add_custom_processes_to_python.obj : error LNK2001: unresolved external symbol "public: __cdecl Kratos::ShellToSolidShellProcess<4>::ShellToSolidShellProcess<4>(class Kratos::ModelPart &,class Kratos::Parameters)" (??0?$ShellToSolidShellProcess@$03@Kratos@@QEAA@AEAVModelPart@1@VParameters@1@@Z)
2>add_custom_processes_to_python.obj : error LNK2001: unresolved external symbol "public: __cdecl Kratos::ShellToSolidShellProcess<3>::ShellToSolidShellProcess<3>(class Kratos::ModelPart &,class Kratos::Parameters)" (??0?$ShellToSolidShellProcess@$02@Kratos@@QEAA@AEAVModelPart@1@VParameters@1@@Z)
2>D:\Kratos-FeatureElement\cmake_build\applications\StructuralMechanicsApplication\Release\KratosStructuralMechanicsApplication.pyd : fatal error LNK1120: 2 unresolved externals
2>Done building project "KratosStructuralMechanicsApplication.vcxproj" -- FAILED.
1>d:\kratos-featureelement\kratos\geometries\tetrahedra_3d_4.h(688): error C2039: '_copysign': is not a member of 'std'
1>c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.13.26128\include\ctime(9): note: see declaration of 'std'
1>d:\kratos-featureelement\kratos\geometries\tetrahedra_3d_4.h(669): note: while compiling class template member function 'double Kratos::Tetrahedra3D4<Kratos::Node<3,Kratos::Dof<double>>>::VolumeToEdgeLengthQuality(void) const'
1>d:\kratos-featureelement\applications\meshingapplication\custom_utilities\local_refine_tetrahedra_mesh.hpp(251): note: see reference to class template instantiation 'Kratos::Tetrahedra3D4<Kratos::Node<3,Kratos::Dof<double>>>' being compiled
1>d:\kratos-featureelement\external_libraries\pybind11\cast.h(1500): note: see reference to class template instantiation 'pybind11::detail::descr<8>' being compiled
1>d:\kratos-featureelement\external_libraries\pybind11\cast.h(1498): note: see reference to class template instantiation 'pybind11::detail::descr<5>' being compiled
1>d:\kratos-featureelement\external_libraries\pybind11\cast.h(1053): note: see reference to class template instantiation 'pybind11::detail::descr<7>' being compiled
1>Done building project "KratosMeshingApplication.vcxproj" -- FAILED.
```

**Comment by PhilippeBussetta** [comment]
I have still a error with `tetrahedra_3d_4`

```
1>add_custom_utilities_to_python.cpp
1>Info: Boost.Config is older than your compiler version - probably nothing bad will happen - but you may wish to look for an update Boost version.  Define BOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE to suppress this message.
1>d:\kratosoff\kratos\geometries\tetrahedra_3d_4.h(688): error C2039: '_copysign': is not a member of 'std'
1>c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.13.26128\include\ctime(9): note: see declaration of 'std'
1>d:\kratosoff\kratos\geometries\tetrahedra_3d_4.h(669): note: while compiling class template member function 'double Kratos::Tetrahedra3D4<Kratos::Node<3,Kratos::Dof<double>>>::VolumeToEdgeLengthQuality(void) const'
1>d:\kratosoff\applications\meshingapplication\custom_utilities\local_refine_tetrahedra_mesh.hpp(251): note: see reference to class template instantiation 'Kratos::Tetrahedra3D4<Kratos::Node<3,Kratos::Dof<double>>>' being compiled
1>d:\kratosoff\external_libraries\pybind11\cast.h(1500): note: see reference to class template instantiation 'pybind11::detail::descr<8>' being compiled
1>d:\kratosoff\external_libraries\pybind11\cast.h(1498): note: see reference to class template instantiation 'pybind11::detail::descr<5>' being compiled
1>d:\kratosoff\external_libraries\pybind11\cast.h(1053): note: see reference to class template instantiation 'pybind11::detail::descr<7>' being compiled
1>Done building project "KratosMeshingApplication.vcxproj" -- FAILED.
```

**Comment by loumalouomega** [comment]
Any idea @roigcarlo?

**Comment by roigcarlo** [comment]
Hi, the copysign error has been solved here #1856 . I will look for the others.

Edit: Ups, was not merged, now it is sorry.
